extensions/amp-story/1.0/media-pool.js

```diff
@@ -735,9 +713,17 @@ export class MediaPool {
     return this.enqueueMediaElementTask_(poolMediaEl, new PauseTask())
         .then(() => {
           if (rewindToBeginning) {
-            this.enqueueMediaElementTask_(
-                /** @type {!HTMLMediaElement} */ (poolMediaEl),
-                new RewindTask());
+            if (desktopState) {
+              this.timer_.delay(() => {
+                this.enqueueMediaElementTask_(
+                  /** @type {!HTMLMediaElement} */ (poolMediaEl),
+                  new RewindTask());
+              }, 300);
```

Comment:
(didn't review the whole pr)

Is there a way we can tightly couple this to where it's actually used?  If not, can we at least leave a comment as to why we're delaying by 300ms (and where that 300ms is defined)?